docs/doxygen-output/html/classrrllvm_1_1ModelDataIRBuilder.html

```diff
@@ -281,10 +281,10 @@
 static const char *&#160;</td><td class="memItemRight" valign="bottom"><b>csr_matrixName</b> = &quot;rr_csr_matrix&quot;</td></tr>
 <tr class="separator:a93563e90ab8b9113af95eb6fc3fe7712"><td class="memSeparator" colspan="2">&#160;</td></tr>
 <tr class="memitem:aa128aa56d7a80229166d0466db3ea013"><td class="memItemLeft" align="right" valign="top"><a id="aa128aa56d7a80229166d0466db3ea013"></a>
-static const char *&#160;</td><td class="memItemRight" valign="bottom"><b>csr_matrix_set_nzName</b> = &quot;rr_csr_matrix_set_nz&quot;</td></tr>
+static const char *&#160;</td><td class="memItemRight" valign="bottom"><b>csr_matrix_set_nzName</b> = &quot;csr_matrix_set_nz&quot;</td></tr>
 <tr class="separator:aa128aa56d7a80229166d0466db3ea013"><td class="memSeparator" colspan="2">&#160;</td></tr>
```

Comment:
It also seems odd to me that these functions would be renamed to *remove* the 'rr', while the previous functions (arccot_negzero) were renamed to *add* it.